src/main/java/com/api/customer/constants/ErrorMessages.java

```diff
@@ -0,0 +1,12 @@
+package com.api.customer.constants;
+
+/**
+ * Define error messages.
+ * 
+ * @author thutrang
+ */
+public class ErrorMessages {
+
+    // Error message from resources: api.message.error.id.not.found
+    public static final String ERROR_MESSAGE_CUSTOMER_ID_NOT_FOUND = "api.message.error.id.not.found";
```

Comment:
không cần đặt dài như api.message.error.id.not.found. Vd như idNotFound là được

Reply:
đã xong